packages/expandable/components/Expandable.tsx

```diff
@@ -26,40 +25,59 @@ const Expandable: React.FC<ExpandableProps> = ({
   controlledIsOpen,
   onChange,
   indicatorPosition
-}) => (
-  <Toggle defaultOn={Boolean(isOpen)} on={controlledIsOpen} onToggle={onChange}>
-    {({ on, getTogglerProps }) => (
-      <div>
-        <ResetButton
-          {...getTogglerProps({
-            className: cx(labelClassName, toggler)
-          })}
+}) => {
+  const [open, setOpen] = React.useState(Boolean(isOpen));
+
+  React.useEffect(() => {
+    if (typeof controlledIsOpen !== "undefined") {
+      setOpen(controlledIsOpen);
+    }
+  }, [controlledIsOpen]);
+
+  const handleToggle = () => {
+    const newOpen = !open;
+
+    if (typeof controlledIsOpen === "undefined") {
+      setOpen(newOpen);
+    }
+
+    if (onChange) {
```

Comment:
based on what I could glean, these two props aren't mutually exclusive. For instance, you might have an onChange even if the parent isn't controlling the toggle. Also, if the parent _is_ controlling the toggle, it would need an onChange callback to know when to change its own state